src/scripts/main.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
 import './_select';
-import './_where-to-find';
+import { addListenerForGalleryButtons } from './_button-set-gallery';
 import './_card-animation';
 import { addScrollAnimation } from './_scroll-animation';
 import './_change-content-animation';
 import './_set-year';
 
+addListenerForGalleryButtons();
```

Comment:
put functions inside `DOMContentLoaded` or use `DOMContentLoaded` inside the function